src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,185 @@
+
+<figure class="w-figure w-figure--center">
+  <img src="weather_app.jpg" alt="An example of a frosted glass effect.">
+  <figcaption class="w-figcaption">
+    An example of a frosted glass effect. <a href="https://dribbble.com/shots/733714-Weather-App?list=tags&tag=android" target="_blank">Source</a>.
+  </figcaption>
+</figure>
+
+Historically, these techniques were difficult to implement on the web, requiring less than perfect [hacks or workarounds](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter). In recent years both [Safari](https://webkit.org/blog/3632/introducing-backdrop-filters/) and Edge have provided these capabilities through the `background-filter` (and alternatively, the `-webkit-backdrop-filter`) property which dynamically blends foreground and background colors based on filter functions. Now Chrome supports `background-filter`, starting in version 76.
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-kitchen_sink.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    A demonstration of the filter functions for <code>backdrop-filter</code>. Try the example on <a href="https://codepen.io/robinrendle/pen/LmzLEL" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+## Basics
+
+- The `backdrop-filter` property applies one or more filters to an element, changing the appearance of anything behind the element.
+- The overlaying element must be at least partially transparent.
+- The overlaying element will get a new stacking context.
+
+{% Aside 'caution' %}
+`backdrop-filter` may harm performance. Test it before deploying.
+{% endAside %}
+
+CSS `backdrop-filter` applies one or more filters to the backdrop of an element, leaving the contents unchanged. Some people find its name confusing. Is it changing the background? It does not. Rather, it alters the appearance of whatever is seen through it. It almost goes without saying that if the element has no transparency, that is, if its value is set to `none` there will be no effect on the background.
+
+The following example illustrates this. For comparison, the image on the left shows how overlapping elements would be rendered if `backdrop-filter` were not used or supported. The image on the right applies `backdrop-filter` and `opacity`. Here's the CSS declaration that defines this.
+
+```css
+.frosty-glass-pane {
+  opacity: .9;
+  backdrop-filter: blur(2px);
+}
+```
+
+<figure class="w-figure w-figure--center">
+  <img src="backdrop-filter.png" alt="The item on the right uses <code>backdrop-filter<code>. The one on the left does not.">
```

Comment:
This still isn't ideal for accessibility since there's text in the image. See [WCAG SC 1.4.5](https://www.w3.org/TR/WCAG21/#images-of-text). I can break this up and put it in a columns widget if that works.

Reply:
Go ahead.

Reply:
Done.